src/routes/const.ts

```diff
@@ -0,0 +1 @@
+export const defaultRoute = "/patients";
```

Comment:
@SteveGT96 I would suggest to move this const to /customization/landingPage.ts, because it can be customized and is related to route configuration.